src/xy/xySortX.ts

```diff
@@ -24,7 +24,7 @@ export function xySortX(data: DataXY): DataXY<Float64Array> {
     }
   }
 
-  const xyObject = (x as number[])
+  const xyObject = Array.from(x)
     .map((val, index) => ({
```

Comment:
```suggestion
  const xyObject = Array.from(x, (val, index) => ({
```

More performant than two calls, and more concise than a loop.

Reply:
JSBench:
- with sort: https://jsbench.me/xblxm9freo/1
- without sort: https://jsbench.me/xblxm9freo/2
In this function it seems sorting is the limiting step